react-app/src/tests/__tests__/Reducers/TestReducers.test.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable max-len */
+import expect from 'expect';
+
+import * as reducers from '../../../reducers/Reducers';
+import * as actions from '../../../actions/Actions';
+import { IMovie } from '../../../components/molecules/MovieCard/IMovie';
+
+const movieObj: IMovie = {
+  id: 1,
+  title: 'Moana',
+  subtitle: 'Moana',
+  description: 'description',
+  rate: 5,
+  duration: 30,
+  yearOfRelease: '2020',
+  genre: ['Horror'],
+};
+
+describe('post reducer', () => {
+  it('should return the MODAL_VISIBILITY state', () => {
```

Comment:
I don't want to be picky, but I'd say that following [AAA pattern](https://freecontent.manning.com/making-better-unit-tests-part-1-the-aaa-pattern/) could've improved your tests readability and maintainability.

Reply:
Update only one test with AAA-pattern.
https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80